### PR TITLE
Fix #1755: Spring boot enricher does not produce a proper heath check…

### LINKED ISSUE #1755: Spring boot enricher does not produce a proper heath check and liveness check path when "/" is used.
# Description
Spring boot enricher does not produce a proper heath check and liveness check path when "/" is used in some path configurations, as an example using `management.endpoints.web.base-path=/` in spring boot 2 application.properties results in the evaluation of `//health` path and should have been `instead`.

### Info

* f-m-p version : 4.3.1
* Maven version (`mvn -v`) : 
```
Apache Maven 3.5.2 (138edd61fd100ec658bfa2d307c43b76940a5d7d; 2017-10-18T09:58:13+02:00)
Maven home: /home/valdar/bin/apache-maven-3.5.2
Java version: 1.8.0_212, vendor: Oracle Corporation
Java home: /usr/lib/jvm/java-1.8.0-openjdk-1.8.0.212.b04-0.fc30.x86_64/jre
Default locale: en_GB, platform encoding: UTF-8
OS name: "linux", version: "5.1.18-300.fc30.x86_64", arch: "amd64", family: "unix"
```

* If it's a bug, how to reproduce :
just use `management.endpoints.web.base-path=/` in a spring boot 2 project and try to `fabric8:deploy`.

### PR DESCRIPTION
**Pull request by valdar**
… and liveness check path when / is used.

### CODE REVIEW

**Comment by dev-gaur** [comment]
fixes #1755

**Comment by valdar** [comment]
@dev-gaur PR updated with changelog.

File: enricher/fabric8/src/test/java/io/fabric8/maven/enricher/fabric8/AbstractSpringBootHealthCheckEnricherTestSupport.java [hunk 1]
```diff
@@ -137,6 +169,42 @@ public void testWithServerPortAndManagementPortAndManagementContextPath() {
...
+        props.put(propertyHelper.getActuatorBasePathPropertyKey(), "/");
+
+        Probe probe = enricher.buildProbe(props, 10, null, null, 3, 1);
+        assertNotNull(probe);
+        assertNotNull(probe.getHttpGet());
+        assertEquals("/health", probe.getHttpGet().getPath());
+        assertEquals(8383, probe.getHttpGet().getPort().getIntVal().intValue());
+    }
+
+    //////TODO:
```
Comment (rohanKanojia) [inline enricher/fabric8/src/test/java/io/fabric8/maven/enricher/fabric8/AbstractSpringBootHealthCheckEnricherTestSupport.java]: Did you forget to add some todo here?

Comment (valdar) [reply enricher/fabric8/src/test/java/io/fabric8/maven/enricher/fabric8/AbstractSpringBootHealthCheckEnricherTestSupport.java]: Actually, I forgot to remove it :D

**Comment by rohanKanojia** [comment]
@valdar : Thanks :+1:

**Pull request merged**